models/poe_res_config_settings.py

```diff
@@ -0,0 +1,6 @@
+from odoo import models, fields
+
+class ResConfigSettingsEnhancement(models.TransientModel):
+    _inherit = 'res.config.settings'
+    
+    default_lifespan = fields.Integer(string="Lifespan", default_model='purchase.order')
```

Comment:
Please try to change the value of this field on Settings and you'll see this is not enough to define a field like this in `res.config.settings`